pkgs/by-name/z3/z3/package.nix

```diff
@@ -97,21 +101,11 @@ stdenv.mkDerivation (finalAttrs: {
       "dev"
       "python"
     ]
-    ++ lib.optional javaBindings "java"
-    ++ lib.optional ocamlBindings "ocaml";
+    ++ lib.optionals javaBindings [ "java" ]
+    ++ lib.optionals ocamlBindings [ "ocaml" ];
 
   passthru = {
-    updateScript = nix-update-script {
-      extraArgs =
```

Comment:
We should keep the version regex in the extraArgs - it's how the update script knows what version to pull out of the release name, IIRC.

Reply:
Yes, it used to, but after this the only major version kept in nixpkgs is the latest one anyways so I just removed it so the bot can also update `z3` to `4.16` for example in the future.

Reply:
I think the match group there is the particularly important part (in addition to how lenient the regex is, obviously) - the tags all start z3-

Reply:
Ah, makes sense (well not the part where z3 has a weird version format) but yeah I guess it's needed still...